app/shared/src/commonMain/kotlin/ui/subject/episode/EpisodeVideo.kt

```diff
@@ -171,13 +171,18 @@ internal fun EpisodeVideoImpl(
                 playerState.seekTo(playerState.currentPositionMillis.value + it * 1000)
             }
             val videoPropertiesState by playerState.videoProperties.collectAsState()
-            LaunchedEffect(videoPropertiesState) {
-                swipeSeekerState.enabled = if (videoPropertiesState == null) {
-                    false
-                } else {
-                    videoPropertiesState!!.durationMillis != 0L
+            val enableSwipeToSeek by remember {
+                derivedStateOf {
+                    with(videoPropertiesState) {
+                        if (this == null) {
+                            false
+                        } else {
+                            this.durationMillis != 0L
+                        }
+                    }
```

Comment:
`with` is stange here

```suggestion
                    videoPropertiesState?.let { it.durationMillis != 0L } ?: false
```